types/Cargo.toml

```diff
@@ -14,17 +14,18 @@ keywords = ["blockchain", "celestia", "lumina"]
 categories = ["encoding", "cryptography::cryptocurrencies"]
 
 [dependencies]
-blockstore = { workspace = true }
-celestia-proto = { workspace = true }
+blockstore.workspace = true
+celestia-proto.workspace = true
 celestia-tendermint = { workspace = true, features = ["std", "rust-crypto"] }
-celestia-tendermint-proto = { workspace = true }
-nmt-rs = { workspace = true }
+celestia-tendermint-proto.workspace = true
+nmt-rs.workspace = true
 
 base64 = "0.22.1"
 bech32 = "0.11.0"
 bytes = "1.6.0"
 cid = { version = "0.11.1", default-features = false, features = ["std"] }
 const_format = "0.2.32"
+cosmrs = "0.21.0"
```

Comment:
I'm in general a bit concerned about cosmrs dependency, is it hardly needed? If so, maybe we should first resolve tendermint issue... :see_no_evil: